### PR TITLE
`README` improvement

### PR DESCRIPTION
**Pull request by grigasp**
Closes https://github.com/iTwin/presentation/issues/751.

- Moved all existing content from `README.md` to `CONTRIBUTING.md`.
- In the updated `README.md`, added a description of what problems the library solves and what packages it consists of.

@iTwin/itwinjs-core-presentation, please all have a look at let me know if you think there's anything else worth adding.

### CODE REVIEW

**Review by saskliutas — COMMENTED** [review]
Diagram looks bad using dark mode:


image

**Comment by grigasp** [comment]
> Diagram looks bad using dark mode:

🤦

**Review by saskliutas — APPROVED** [review]
Looks good to me. Maybe we should include link to `CONTRIBUTING.md` in `README.md` so it would be easier to find?

File: media/presentation-packages.png
Comment (JonasDov) [inline media/presentation-packages.png:1]: This picture:
image
 contains a sentence: `Some of they may be useful for consumers as well.` It should be changed to `Some of them may be useful for consumers as well.`

**Review by JonasDov — APPROVED** [review]

**Review by MartynasStrazdas — COMMENTED** [review]
We might want to include a link to the learning folder for hierarchies to ensure it's easily accessible and to encourage usage. Maybe a section for it if we expect more packages to have learning-snippets or a link under the description of the package.

File: media/presentation-packages.png
Comment (grigasp) [reply media/presentation-packages.png:1]: fixed

**Comment by grigasp** [comment]
> Maybe we should include link to `CONTRIBUTING.md` in `README.md` so it would be easier to find?

Do you think it's hard to find now? I feel like if you're already in a git repo, reading README and willing to contribute, then you're probably aware of other common markdowns like LICENCE.md and CONTRIBUTING.md. Just don't want to litter our front page with obvious stuff...

**Comment by grigasp** [comment]
> We might want to include a link to the learning folder for hierarchies to ensure it's easily accessible and to encourage usage. Maybe a section for it if we expect more packages to have learning-snippets or a link under the description of the package.

The learning folder has no entry point markdown, so it's probably not the best place to land someone looking for learning material. Out entry point to learning material is the [Learning section in the README](https://github.com/iTwin/presentation/blob/master/packages/hierarchies/README.md#learning). I updated the package links to target READMEs, so not sure we want to add another link pointing to the same readme, but just specific section of it. What do you think?

**Comment by MartynasStrazdas** [comment]
> > We might want to include a link to the learning folder for hierarchies to ensure it's easily accessible and to encourage usage. Maybe a section for it if we expect more packages to have learning-snippets or a link under the description of the package.
> 
> The learning folder has no entry point markdown, so it's probably not the best place to land someone looking for learning material. Out entry point to learning material is the [Learning section in the README](https://github.com/iTwin/presentation/blob/master/packages/hierarchies/README.md#learning). I updated the package links to target READMEs, so not sure we want to add another link pointing to the same readme, but just specific section of it. What do you think?

I think if they are going to open the package README they will notice the learning section, no need to add another link to the same file.

**Review by MartynasStrazdas — APPROVED** [review]

**Pull request merged**